Infrastructure/Resources/New-StorageAccountContainer.ps1

```diff
@@ -30,7 +30,11 @@ Param(
     [Parameter(Mandatory = $true)]
     [String]$Name,
     [Parameter(Mandatory = $true)]
-    [String[]]$ContainerName
+    [String[]]$ContainerName,
+    [Parameter(Mandatory= $true)]
+    [ValidateSet("Container", "Blob", "Off")]
+    [String[]]$ContainerPermission
```

Comment:
Should this be an array?

Reply:
It should also have a default value of Off to make sure exisint implementations don't break

Reply:
done